tests/cloud_test_logic/delete_all_cloud_test_indexes.py

```diff
@@ -38,10 +41,13 @@ def delete_all_test_indices(wait_for_readiness=False):
     print("Marqo Cloud deletion responses:")
     for index_name in indices_to_delete:
         index = client.index(index_name)
```

Comment:
wrap in try-except block and add retries as transient network errors could fail this function

Reply:
done